abis_mapping/templates/survey_site_data/schema.json

```diff
@@ -0,0 +1,142 @@
+      "example": "-34.036",
+      "type": "number",
+      "format": "default",
+      "constraints": {
+        "required": false,
+        "minimum": -90,
+        "maximum": 0
+      }
+    },
+    {
+      "name": "decimalLongitude",
+      "title": "Decimal Longitude",
+      "description": "The geographic longitude (in decimal degrees, using the spatial reference system given in geodeticDatum) of the geographic origin of a Site. Positive values are east of the Greenwich Meridian, negative values are west of it. Legal values lie between 0 and 180, inclusive for the BDR use case.",
+      "example": "146.363",
+      "type": "number",
+      "format": "default",
+      "constraints": {
+        "required": false,
+        "minimum": 0,
+        "maximum": 180
+      }
+    },
+    {
+      "name": "footprintWKT",
+      "title": "Footprint WKT",
+      "description": "A Well-Known Text (WKT) representation of the shape (footprint, geometry) that defines the Site. A Site may have both a point-radius representation and a footprint representation, and they may differ from each other.",
+      "example": "LINESTRING (146.363 -34.036, 146.363 -34.037)",
+      "type": "wkt",
+      "format": "default",
+      "constraints": {
+        "required": false
+      }
+    },
+    {
+      "name": "geodeticDatum",
+      "title": "Geodetic Datum",
+      "description": "The geodetic datum, or spatial reference system (SRS) upon which the geographic coordinates given for the Site are based.",
+      "example": "WGS84",
+      "type": "string",
+      "format": "default",
+      "constraints": {
+        "required": true,
+        "enum": [
+          "AGD84",
+          "EPSG:4203",
+          "GDA2020",
+          "EPSG:7844",
+          "GDA94",
+          "EPSG:4283",
+          "WGS84",
+          "EPSG:4326"
+        ]
+      }
+    },
+    {
+      "name": "coordinateUncertaintyInMeters",
+      "title": "Coordinate Uncertainty in Meters",
+      "description": "The horizontal distance (in metres) from the given decimalLatitude and decimalLongitude describing the smallest circle containing the whole of the Site. Leave the value empty if the uncertainty is unknown, cannot be estimated, or is not applicable (because there are no coordinates). Zero is not a valid value for this term.",
+      "example": "50",
+      "type": "integer",
+      "format": "default",
+      "constraints": {
+        "required": false,
+        "minimum": 1
+      }
+    },
+    {
+      "name": "siteVisitStart",
+      "title": "Site Visit Start",
+      "description": "The temporal start of when the Site was being used to collect data for the survey. Expected values include date, dateTime, dateTimeStamp.",
+      "example": "2016-02-28",
+      "type": "timestamp",
+      "format": "default",
+      "constraints": {
+        "required": true
```

Comment:
I have a doubt of this being required but it is not the place to discuss about it.